span_test.go

```diff
@@ -3,6 +3,7 @@ package basictracer
 import (
 	"testing"
 
+	opentracing "github.com/opentracing/opentracing-go"
```

Comment:
is the `opentracing` on the left needed? (I thought it happened implicitly since that's the name of the package)